packages/admin-ui/src/pages/wifi/components/localProxying/LocalProxying.tsx

```diff
@@ -7,19 +7,43 @@ import { ReqStatus } from "types";
 // Own components
 import { confirm } from "components/ConfirmDialog";
 import { withToast } from "components/toast/Toast";
+import { rootPath as installedRootPath } from "pages/installer";
 import ErrorView from "components/ErrorView";
 import Loading from "components/Loading";
 import Card from "components/Card";
 import Switch from "components/Switch";
 import { getDappnodeIdentityClean } from "services/dappnodeStatus/selectors";
-import { adminUiLocalDomain, docsUrl } from "params";
+import { adminUiLocalDomain, docsUrl, httpsPortalDnpName } from "params";
 import { StateBadge } from "pages/packages/components/StateBadge";
 import { MdWifi } from "react-icons/md";
 import { parseContainerState } from "pages/packages/components/StateBadge/utils";
+import Alert from "react-bootstrap/esm/Alert";
 import { LocalProxyingStatus } from "common";
+import { NavLink } from "react-router-dom";
 import LinkDocs from "components/LinkDocs";
 
 export function LocalProxying(): JSX.Element {
+  const dnpsRequest = useApi.packagesGet();
```

Comment:
Just inline this logic in the same component. Now you won't show any help text describing this feature. So the text says "You must install the HTTPs Portal to use this feature", but what feature is it?

Reply:
done